README.md

```diff
@@ -208,6 +208,33 @@ oktaOidc.signOutOfOkta(authStateManager, from: self) { error in
 ```
 Sample app [example](https://github.com/okta/samples-ios/blob/master/browser-sign-in/OktaBrowserSignIn/SignInViewController.swift#L62-L74)
 
+### signOut
+
+This method helps to perform mutli-step sign out flow. Provide options that you want to perform and sdk will run them as a batch. Available options are:
+- revokeAccessToken - sdk revokes access token
+- revokeRefreshToken - sdk revokes refresh token
+- signOutFromOkta - sdk calls [`signOutOfOkta`](#signoutofokta)
+- removeTokensFromStorage - sdk removes tokens from the secure storage
+- revokeTokensOptions - revokes access and refresh tokens
+- allOptions - revokes tokens, signs out from Okta and removes tokens from the secure storage
```

Comment:
allOptions - revokes tokens, signs out from Okta, and removes tokens from the secure storage